src/main/resources/reference.conf

```diff
@@ -268,11 +268,17 @@ resourceTypes = {
       add_to_service_perimeter = {
         description = "add this billing-project to a service-perimeter"
       }
+      delete_service_perimeter = {
```

Comment:
It looks to me like in the ticket we decided not to permit the removal of a project from a perimeter.  

If I am wrong, can we make sure the ticket is updated accordingly?  Also, if I am wrong and we _do_ need this permission, then I would rather it be called `remove_from_perimeter` or `delete_project_from_perimeter` or something like that.

Reply:
Ahh, sorry, I forgot when I added delete_service_perimeter but that is not intended. Removed.

Thanks